manual/src/html_processing/src/common.ml.in

```diff
@@ -27,21 +27,23 @@ let ( // ) = Filename.concat
 
 let process_dir = Filename.current_dir_name
 
+let ocaml_version = "@OCAML_VERSION_SHORT@"
+
 (* Output directory *)
-let web_dir = Filename.parent_dir_name // "webman"
+let web_dir = Filename.parent_dir_name // ocaml_version
 
 (* Output for manual *)
-let docs_maindir = web_dir // "manual"
+let docs_maindir = web_dir
 let docs_file = ( // ) docs_maindir
 
 (* Output for API *)
 let api_dir = web_dir // "api"
 
 (* How to go from manual to api *)
-let api_page_url = "../api"
+let api_page_url = "../" ^ ocaml_version ^ "/api"
```

Comment:
Is there a reason why this path is not `./api`?
If this is the case, a comment would be nice.

Reply:
The manual here refers to `htmlman`. Without the above defined path, I get the following error:
```
Fatal error: exception Sys_error("../htmlman/./5.3/api/compilerlibref/index.html: No such file or directory")
```

Reply:
Let's update the comment to reflect that point: `manual` ⇒ `htmlman manual`.

Reply:
The explanation is erroneous in fact, which explains why the path `htmlman/../5.3/api` doesn't correspond to anything.
The real issue here is that the `Process_manual.get_xfile` filters the api index using the `../*` pattern when looking for files to add to the table of contents.
Using `../ocaml_version/api` rather than just `api` allow to still match this pattern while keeping the correct api.
In other words the better fix is to keep `api_page_url="api"` and fix `get_xfile` with
```diff
-             not (starts_with ".." rf) &&
+             not (starts_with "api" rf) &&
```